c2corg_api/views/user.py

```diff
@@ -116,20 +120,128 @@ def post(self):
             log.warning('Error persisting user', exc_info=True)
             raise HTTPInternalServerError('Error persisting user')
 
-        # also create a version for the profile
-        DocumentRest.create_new_version(user.profile, user.id)
```

Comment:
Ok. Maybe it would make sense to delay the creation of the user-profile until the account is confirmed. Otherwise un-confimed users might show up in the search and listings.